.github/workflows/latest_dependency_checker.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         source venv_core/bin/activate
         python -m pip install --upgrade pip
         python -m pip install -e .
```

Comment:
I think this line might be unnecessary now.

Reply:
are you referring to only line `python -m pip install -e .`

Reply:
Yes, I think the `python -m pip instal .[test]` command should also install the library whereas before the extra `python -m pip install -r test-requirements.txt` command was needed to add the test requirements to the base library install.

Reply:
sounds good, updated